lib/vm/src/function_env.rs

```diff
@@ -1,7 +1,11 @@
+use derivative::Derivative;
 use std::any::Any;
 
 /// Underlying FunctionEnvironment used by a `VMFunction`.
+#[derive(Derivative)]
+#[derivative(Debug)]
```

Comment:
What do we gain by making this Debug?

Reply:
It's useful for debugging.

In general all public types should be `Debug` - that's also in the Rust API guidelines.

Reply:
I don't think any VM structure should be debuggable to be honest. It's not meant to be used by anyone outside of wasmer

Reply:
Removing this makes `StoreObjects` non `Debug` too, wich contains many usefull Debug information. So it would be better to keep it that way.